src/virtual_secondary/virtualsecondary.cc

```diff
@@ -57,10 +55,13 @@ void VirtualSecondaryConfig::dump(const boost::filesystem::path& file_full_path)
   Json::Value root;
   root[Type].append(json_config);
 
-  Json::StyledStreamWriter json_writer;
+  Json::StreamWriterBuilder json_bwriter;
+  json_bwriter["indentation"] = "\t";
+  std::unique_ptr<Json::StreamWriter> const json_writer(json_bwriter.newStreamWriter());
+
   boost::filesystem::create_directories(file_full_path.parent_path());
   std::ofstream json_file(file_full_path.string());
-  json_writer.write(json_file, root);
+  json_writer->write(root, &json_file);
   json_file.close();
 }
```

Comment:
Is anyone using this `dump` function? It doesn't look like it.

Reply:
Seems to be used in tests: https://github.com/advancedtelematic/aktualizr/blob/dd73a67eb42e60727527b72cc2285170dd842c66/tests/uptane_test_common.h#L80

Reply:
Ah, there it is! Sorry, missed that somehow.